scripts/server.ts

```diff
@@ -17,7 +17,7 @@ const parsed = JSON.parse(readFileSync("./event.json", "utf8"))
 
 const requestHandler = async (
   req: IncomingMessage,
-  res: ServerResponse
+  res: ServerResponse,
```

Comment:
guessing this might be some added noise from a prettier/linting rule? hanging commas?

Reply:
Correct, eslint and prettier have updated standard compared to the previous versions. We can look at overwriting the default rules if we want to remove these changes from this PR

Reply:
It might have made it a bit more digestible but I'm through it now :p And we don't have a standard, as an org, on hanging commas so I don't think it's the effort to revert